common/source/docs/common-use-cases-and-applications.rst

```diff
@@ -21,6 +21,17 @@ Use Cases
 
 .. image:: ../../../images/case-tractor.jpg
 
+
+- `Safe flights by agile collision avoidance <https://discuss.ardupilot.org/t/cmu-nrec-falco-drone-safe-flights-by-agile-collision-avoidance/83300>`__
```

Comment:
After looking at the built page, this tilte needs working a bit....am going to merge it and then tweak it....thanks

Reply:
Hi @Hwurzburg, after talking to the team, I think we can change this the title to "Autonomous drone with agile collision avoidance"